cranelift-codegen/src/isa/x86/abi.rs

```diff
@@ -361,14 +367,27 @@ fn fastcall_prologue_epilogue(func: &mut ir::Function, isa: &dyn TargetIsa) -> C
         func.signature.returns.push(csr_arg);
     }
 
+    let mut cfa_state = CFAState {
+        reg: RU::rsp as RegUnit,
+        offset: word_size as isize,
+        position: -(word_size as isize),
```

Comment:
These members names are a bit unclear: my understanding is that `position` points to the return address position, while `reg` / `offset` are information telling what's the current register + offset pointing to the call frame. Is that correct? If so, we should probably rename these.